platform/docs/docs/platform/services/customization-service/sampleCustomizations.tsx

```diff
@@ -0,0 +1,1154 @@
+import React from 'react';
+import measurementLabelsImage from '../../../assets/img/measurement-labels-auto.png';
+import seriesSortImage from '../../../assets/img/seriesSort.png';
+import windowLevelPresetsImage from '../../../assets/img/windowLevelPresets.png';
+import colorbarImage from '../../../assets/img/colorbarImage.png';
+import segmentationTableModeImage from '../../../assets/img/segmentationTableModeImage.png';
+import segmentationTableModeImage2 from '../../../assets/img/segmentationTableModeImage2.png';
+import segmentationShowAddSegmentImage from '../../../assets/img/segmentationShowAddSegmentImage.png';
+import layoutSelectorCommonPresetsImage from '../../../assets/img/layoutSelectorCommonPresetsImage.png';
+import layoutSelectorAdvancedPresetGeneratorImage from '../../../assets/img/layoutSelectorAdvancedPresetGeneratorImage.png';
+
+import segDisplayEditingTrue from '../../../assets/img/segDisplayEditingTrue.png';
+import segDisplayEditingFalse from '../../../assets/img/segDisplayEditingFalse.png';
+
+import thumbnailMenuItemsImage from '../../../assets/img/thumbnailMenuItemsImage.png';
+import studyMenuItemsImage from '../../../assets/img/studyMenuItemsImage.png';
+
+export const viewportOverlayCustomizations = [
+  {
+    id: 'viewportOverlay.topRight',
+    description: 'Defines the items displayed in the top-right overlay of the viewport.',
+    default: [],
+    configuration: `
+window.config = {
```

Comment:
Shouldn't these be window.config.customizationService = [...]
Otherwise you replace the entire window.config, which seems a bit excessive.